docs/main/06-kratix-cli/00_intro.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Introduction
+description: Introduction to Kratix and how to get started with the Kratix CLI
+sidebar_label: Introduction
+id: intro
+---
+
+The Kratix CLI is a tool designed to help you build promises and manage your Kratix
+installation.
+
+## Installation
+
+To install the Kratix CLI, go to the [releases
+page](https://github.com/syntasso/kratix-cli/releases), download the latest release of
+the binary for your platform.
+
+Once you have downloaded the binary, you can install it by moving it to a directory in
+your PATH.
+
+To verify your installation, run:
+
+```shell-session
+kratix --version
+```
+
+### Enable shell autocompletion
+
+To configure your shell to load the CLI bash completions, add to your profile:
+
+```shell-session
+. <(kratix completion bash)
```

Comment:
I get an error with `source <(kratix completion bash)`. Maybe because of my environment 🤔
```
/dev/fd/16:type:332: bad option: -t
```

Reply:
You are probably using `zsh` instead of bash 👀 will add a line there to clarify